src/lib/MuncherSettings.js

```diff
@@ -282,8 +282,8 @@ const MuncherSettings = {
       {
         name: "use-chris-premades",
         isChecked: game.settings.get(SETTINGS.MODULE_ID, "character-update-policy-use-chris-premades"),
-        title: "Use effects from Chris's Premades module?",
-        description: `Requires Chris's Premades ${MuncherSettings.getInstalledIcon("chrisInstalled")} module. These will replace any effects created by DDB Importer.`,
+        title: "Use ActiveEffects from Chris's Premades module for high automation?",
+        description: `Requires Chris's Premades ${MuncherSettings.getInstalledIcon("chrisInstalled")} module. These will replace any ActiveEffects created by DDB Importer and are recommended for games aiming for high automation.`,
```

Comment:
```suggestion
        description: `Requires Chris's Premades ${MuncherSettings.getInstalledIcon("chrisInstalled")} module. These will replace any ActiveEffects created by DDB Importer.`,
```

Reply:
If CPR and all its dependencies are required, then it is aimed at high automation setups. Is this not correct?

Reply:
I reads like I'm recommending Chris' module for games with high automation.

Reply:
Was that not the intent?